strawberry/utils/inspect.py

```diff
@@ -104,6 +114,14 @@ def get_specialized_type_var_map(cls: type, *, include_type_vars: bool = False):
 
     type_var_map = {}
 
+    # only get type vars for base generics (ie. Generic[T]) and for strawberry types
+
+    orig_bases = [
+        b
+        for b in orig_bases
+        if hasattr(b, "_type_definition") or type(b) is _GenericAlias
+    ]
```

Comment:
@bellini666 does this make sense?

this ignores all the generics on non strawberry types

Reply:
If I understood this correctly, this is to fix that issue reported on discord, right?

I think this makes sense as non strawberry generics in the inheritance shouldn't create different types.